### PR TITLE
Feat/nav bar

### PR DESCRIPTION
**Pull request by V-M-AISHNA**

### CODE REVIEW

**Comment by V-M-AISHNA** [comment]
image
image
image
image

File: Airbnb/index.html [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+                                <div class="filter-text">
+                                    <p>Filters</p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
```
Comment (vonnue-anusha) [inline Airbnb/index.html]: remove spacing

File: Airbnb/index.html [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+    </nav>
+    <main>
+        <div class="main-section">
```
Comment (vonnue-anusha) [inline Airbnb/index.html]: remove this and give main-section styles to main

File: Airbnb/index.html [hunk 3]
```diff
@@ -0,0 +1,399 @@
...
+            <div class="display-total-container">
+                <div class="display-total-block">
```
Comment (vonnue-anusha) [inline Airbnb/index.html]: Are both containers needed?

File: Airbnb/index.html [hunk 4]
```diff
@@ -0,0 +1,399 @@
...
+                            <p>Display total before taxes</p>
+                        </div>
+                    </div>
+                    <div class="toggle-button">
+                        <div class="button-nav">
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="card-group">
```
Comment (vonnue-anusha) [inline Airbnb/index.html]: change class name to cards-container

**Review by vonnue-anusha — CHANGES_REQUESTED** [review]

**Comment by V-M-AISHNA** [comment]
image
image

**Pull request merged**